tests/kernel/mem_protect/syscalls/prj.conf

```diff
@@ -3,3 +3,5 @@ CONFIG_TEST_USERSPACE=y
 CONFIG_TIMESLICING=y
 CONFIG_TIMESLICE_SIZE=10
 CONFIG_APPLICATION_DEFINED_SYSCALL=y
+CONFIG_INIT_STACKS=y
+CONFIG_THREAD_STACK_INFO=y
```

Comment:
Also, this is probably not needed, since this test requires USERSPACE, which in all ARCHes (ARC, ARM, X86) enables THREAD_STACK_INFO.